### PR TITLE
Change criteria for stable component (no stable candidate anymore).

### PR DESCRIPTION
**Pull request by artursouza**
Signed-off-by: Artur Souza <artursouza.ms@outlook.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

### CODE REVIEW

File: daprdocs/content/en/operations/components/certification-lifecycle.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ All components start at the Alpha stage.
 - The component must have component [certification tests](#certification-tests) validating functionality and resiliency
 - The component is maintained by Dapr maintainers and supported by the community
 - The component is well documented and tested
+- The component has been available as Alpha or Beta for at least 1 minor version release of Dapr runtime prior
```
Comment (berndverst) [inline daprdocs/content/en/operations/components/certification-lifecycle.md:50]: Please see https://github.com/dapr/dapr/issues/4970. Once that issue is resolved we should document the requirements listed there.

**Review by berndverst — CHANGES_REQUESTED** [review]
Depends on https://github.com/dapr/dapr/issues/4970

**Review by yaron2 — APPROVED** [review]

File: daprdocs/content/en/operations/components/certification-lifecycle.md (hunk 1 above)
Comment (artursouza) [reply daprdocs/content/en/operations/components/certification-lifecycle.md:50]: Agreed.

**Comment by berndverst** [comment]
We should also add a page with all the certification test details and link to it. I would not make this PR on its own. Please move the entire outcome from https://github.com/dapr/dapr/issues/4970 into the docs.

**Pull request merged**